Classes/Service/Tika/AppService.php

```diff
@@ -61,7 +62,7 @@ protected function initializeService() {
 	public function getTikaVersion() {
 		$tikaCommand = CommandUtility::getCommand('java')
 			. ' -Dfile.encoding=UTF8' // forces UTF8 output
-			. ' -jar ' . escapeshellarg(GeneralUtility::getFileAbsFileName($this->configuration['tikaPath'], FALSE))
+			. ' -jar ' . ShellUtility::escapeShellArgument(GeneralUtility::getFileAbsFileName($this->configuration['tikaPath'], FALSE))
```

Comment:
Well the tika file was escaped previously using escapeshellargs.
If we keep escapeshellargs then I think we should also use the Utility as that one is more generic.